src-tauri/src/utils.rs

```diff
@@ -77,3 +80,16 @@ pub(crate) fn custom_log_dir(id: &str) -> PathBuf {
 
     path.expect("log path generation should succeed")
 }
+
+// TODO: current implementation works only on unix like systems
+#[tauri::command]
+pub(crate) fn create_file(path: &str, content: String) {
+    let mut file = File::create(path).expect("can't create file");
+    file.write_all(content.as_bytes())
+        .expect("couldn't write file");
+    let metadata = file.metadata().expect("can't get metadata");
```

Comment:
```suggestion
        file.metadata()
        .unwrap_or_else(|why| panic!("couldn't get metadata, because: {why}"))
        .permissions()
        .set_mode(0o644);
```

I think this would be more idiomatic, we don't need to store intermediate steps in variables. 
`assert` was a nice touch, thanks for that! We should probably use these in the tests when we have them.